docs/content/tools/_index.md

```diff
@@ -57,7 +57,7 @@ Valid OSCAL is open-ended in application. Some of the tools described provide va
 
 | Name | Provider/Developer | Description | Type |
 |:---|:---|:---|:---|
-| [Atlasity: Continuous Compliance Automation](https://atlasity.io/partnership/) | C2 Labs | Atlasity CE (release 2.0) runs in any environment and supports the development of OSCAL v1.0 content for Catalogs, Profiles, System Security Plans and Components. Additional detail can be found in this blog post: [Atlasity Delivers Free Tools to Create OSCAL Content](https://www.c2labs.com/post/atlasity-delivers-free-tools-to-create-oscal-content). | community edition |
+| [RegScale (formerly Atlasity)](https://regscale.com) | RegScale (spun out of C2 Labs)  | RegScale Community Edition is a free to use, API-centric real-time Governance, Risk and Compliance (GRC) platform that deploys in any environment, shifting left compliance by integrating with security and compliance tools via API to keep compliance documentation continuously up to date. GRC staff can work in the UI, engineers can write scripts to feed the API, and both human- and machine-readable (OSCAL - all layers supported) content is automatically generated on demand, achieving a continuous Authority to Operate (cATO). Additional information can be found here: [RegScale Delivers Free Platform to Create OSCAL Content](https://regscale.com/oscal). | [Community Edition](https://qa-ce.regscale.com/assets/eula-ce.pdf) |
```

Comment:
The list should be in alphabetical order.

I also agree that "Community License" is a potentially confusing term. It has some pretty non-community terms. IMO its a different Commercial License. A middle way forward would be to do it the way Xacta is done:

```suggestion
| [RegScale (formerly Atlasity)](https://regscale.com) | RegScale (spun out of C2 Labs)  | RegScale Community Edition is a free to use, API-centric real-time Governance, Risk and Compliance (GRC) platform that deploys in any environment, shifting left compliance by integrating with security and compliance tools via API to keep compliance documentation continuously up to date. GRC staff can work in the UI, engineers can write scripts to feed the API, and both human- and machine-readable (OSCAL - all layers supported) content is automatically generated on demand, achieving a continuous Authority to Operate (cATO). Additional information can be found here: [RegScale Delivers Free Platform to Create OSCAL Content](https://regscale.com/oscal). | [license](https://qa-ce.regscale.com/assets/eula-ce.pdf) |
```

Reply:
@akarmel I agree with the changes by @bradh. Please refine this entry.